mitodl/teachersportal
# Added back React hot reloading with fixed version of webpack

#### What are the relevant tickets?

Fixes #326
#### What's this PR do?

Adds hot reloading and upgrades to the version of webpack where `__webpack_public_path__` works for hot reloading.
#### Where should the reviewer start?

webpack.config.dev.js
#### How should this be manually tested?

Make a change that's visible from the browser and make sure that the change shows up properly, without a page refresh happening.
